truetype/truetype.go

```diff
@@ -345,6 +429,50 @@ func (f *Font) Index(x rune) Index {
 	return 0
 }
 
+// Name returns the NameIDCode value of a Font.
+// Returns "" on error or not found.
+func (f *Font) Name(id NameIDCode) string {
+	x, platformID, err := parseSubtables(f.name, "name", 6, 12,
+		func(b []byte) bool {
```

Comment:
Move this to the previous line:

x, platformID, err := parseSubtables(f.name, "name", 6, 12, func(b []byte) bool {
	return NameIDCode(u16(b, 6)) == id
})